charts/private-link/values.yaml

```diff
@@ -0,0 +1,171 @@
+################################################################################
+# Global options
+################################################################################
+global:
+  image:
+    # global image pull policy to use for all container images in the chart
+    # can be overridden by individual image pullPolicy
+    pullPolicy:
+    # global registry to use for all container images in the chart
+    # can be overridden by individual image registry
+    registry: registry.synadia.io
+
+  # global labels will be applied to all resources deployed by the chart
+  labels: {}
+
+################################################################################
+# Private Link Deployment and associated resources
+################################################################################
+
+############################################################
+# image pull secret
+############################################################
+# when enabled, the registry for images in the private link deployment
+# will default to global.image.registry
+imagePullSecret:
+  enabled: false
+  registry:
+  username:
+  password:
+
+  # merge or patch the image pull secret
+  # https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#secret-v1-core
+  merge: {}
+  patch: []
+  # defaults to "{{ include "spl.fullname" $ }}-regcred"
+  name:
+
+############################################################
+# config
+############################################################
+# due to the potentially sensitive nature of some of the configuration
+# options, the entire configuration will be stored in a Kubernetes Secret
+#
+config:
+  # URL for Synadia Control Plane
+  # defaults to Synadia Cloud
+  platformURL:
+  # URL for NATS Server
+  natsURL:
+  # Synadia Cloud or Control Plane Token
+  token:
+  # TLS for connection to NATS Server
+  tls:
+    # set secretName in order to mount an existing secret to dir
+    secretName:
+    dir: /etc/private-link/certs
+    cert: tls.crt
+    key: tls.key
+    ca:
+    # Skip certificate validation
+    insecure: false
+    # merge or patch the tls config
+    merge: {}
+    patch: []
+
+  # merge or patch the config
+  merge: {}
+  patch: []
+
+############################################################
+# deployment
+############################################################
+deployment:
+  replicas: 1
```

Comment:
Should we default replicas to 2?  Also, should we add a Pod Disruption Budget of 1?